proposals/primer/application.bs

```diff
@@ -0,0 +1,353 @@
+<pre class='metadata'>
+Title: Solid Application Interoperability - Application Primer
+Shortname: solid-interop-application-primer
+Level: 1
+Status: w3c/ED
+Group: solid-cg
+URL: https://solid.github.io/data-interoperability-panel/primer/applications.html
+Repository: https://github.com/solid/data-interoperability-panel
+Inline Github Issues: title
+Editor: elf Pavlik
+Abstract: Primer for libraries intented to be used by applications.
+Boilerplate: omit conformance
+</pre>
+
+# Introduction # {#introduction}
+
+This primer was developed along side open source TypeScript implementation [[SAI-JS]]
+
+# Shape Trees # {#shape-trees}
+
+Shape Trees [[SHAPETREES]] allow creating virtual data hierarchies and define which shapes [[SHEX]]
+should be used to validate data. In our examples we will use following Shapes and Shape Trees.
+
+<figure>
+  <pre class=include-code>
+  path: snippets/solidshapes.example/trees/Project.ttl
+  highlight: turtle
+  show: 6-100
+  </pre>
+  <figcaption>Project shape tree</figcaption>
+</figure>
+
```

Comment:
A few bits on the shape tree definitions themselves:

* We typically use `st:` as the prefix for shape trees
* st:validatedBy -> st:shape
* st:ShapeTreeResource -> st:Resource

Reply:
i'll leave prefix `shapetrees` since all the rest of snippets is using it. we can change it in separate PR

property and class changed in 

https://github.com/solid/data-interoperability-panel/pull/170/files#diff-84bfe3fde81d8de560fc14fe09815f5c887a035583bc56d9ff56a7f6eb6bb1ddR143